src/inverseRobot.js

```diff
@@ -25,6 +25,16 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const inverseObj = {};
```

Comment:
Give your variables more meaningful names. E.g. in this case `const inversedRobot = {}`